src/components/AddQuiz.js

```diff
@@ -18,25 +19,29 @@ export default function AddQuiz({ addQuiz }) {
   const handleSubmit = (e) => {
     // prevent default behavior of reloading forms
     e.preventDefault()
+    const uid = newFBKey()
     if (!message) toast.error('Please enter a title')
     else {
-      dbRef.child('quiz').push(
-        {
-          id: uuid4(),
-          title: message,
-          completed: false,
-          status: 'active',
-          isEditing: false,
-          countStudents: 0,
-        },
-        (err) => {
-          if (err) {
-            toast.error(err)
-          } else {
-            toast.success('Quiz Added Successfully')
+      dbRef
+        .child('quiz')
+        .child(uid)
```

Comment:
firebase tip: you can do ``ref.child(`quiz/${uid}`)``, or even ``firebase.database().ref(`quiz/${uid}`)``